src/Juvix/Data/Effect/Fail.hs

```diff
@@ -15,6 +15,12 @@ runFail ::
 runFail = fmap (^? _Right) . runError @() . reinterpret (\Fail -> throw ())
 {-# INLINE runFail #-}
 
+ignoreFail ::
```

Comment:
This doesn't seem to be used, but do you think it's useful to have it?

Reply:
I am using it in #2263 so I think it is fine to keep it here